commands/intro.js

```diff
@@ -1,6 +1,6 @@
 //Call: Slash command intro
 //Sets an intro theme for a user. Youtube link
-const { Interaction } = require('discord.js');
+const { Interaction, IntegrationApplication } = require('discord.js');
```

Comment:
Where are we using IntegrationApplication?

Reply:
Unknown, probably autofilled

Reply:
Remove it then